### PR TITLE
Update contract slightly and open for review

### PR DESCRIPTION
**Pull request by auryn-macmillan**
This PR updates the contract based on [discussions in the Ethereum Magicians forum](https://ethereum-magicians.org/t/erc-3722-poster-a-ridiculously-simple-general-purpose-social-media-smart-contract/6751/68?u=auryn).

The intent is for this to be the final version of the contract, the PR also changes the status to "review".

### CODE REVIEW

**Comment by axic** [comment]
@auryn-macmillan you need to resolve conflicts, see the "Resolve conflicts" button.

**Comment by auryn-macmillan** [comment]
> @auryn-macmillan you need to resolve conflicts, see the "Resolve conflicts" button.

Done! Thanks for the quick reminder.

File: EIPS/eip-3722.md [hunk 1]
```diff
@@ -163,17 +175,16 @@ While there would have been some gas savings if we decided against emitting anyt
```
Comment (wschwab) [inline EIPS/eip-3722.md:175]: There's a mandatory "Test Cases" section that should go here (even though this is a ridiculously simple contract). You can link to the test case in the repo in the Reference Implementation.

Comment (MicahZoltu) [reply EIPS/eip-3722.md:175]: Test Cases and Reference Implementation sections are both optional for ERCs.  Test Cases is required only for consensus changes (Core EIPs).

Comment (wschwab) [reply EIPS/eip-3722.md:175]: oh whoops, sorry about that

**Review by lightclient — APPROVED** [review]

**Pull request merged**